package.json

```diff
@@ -31,5 +31,10 @@
     "babel-cli": "~6.14.0",
     "babel-plugin-add-module-exports": "~0.2.1",
     "babel-preset-es2015": "~6.14.0"
+  },
+  "jspm": {
+    "main": "src/index.js",
```

Comment:
For development purpose you could use the github version and run `npm run compile` to be able to use the transpiled version.
There is no issue with that.

Other than that everyone would used the version offered by npm and nothing else